### PR TITLE
8639: Add tool for extracting movement and direction changes from the change log

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

This PR intends to close https://github.com/cityofaustin/atd-data-tech/issues/8639.

## Current State

Through subsequent meetings and discussion, it has become likely that this branch will continue to evolve.

## Testing

This is a stand-alone, one-time-use app. However, it does provide a solid jumping off point for using the change log to extract the most recent change that has been made to a certain field for a certain table.

The tool's output is a stream of `UPDATE` SQL statements and does not modify any resources itself. Testing can be conducted by spinning up the docker-compose stack, [creating and populating the two databases](https://github.com/cityofaustin/atd-vz-data/blob/movement_direction_data_recovery_tools/atd-toolbox/direction_movement/app/direction_movement_data_recovery.py#L53-L58) from backups found on S3 for the VZDB, and executing the script.

The script is intended to be run on the `vz_mvt_dir_recovery_environment` container which is an expansion of the postgis image via `apt-get` installing [some python libraries](https://github.com/cityofaustin/atd-vz-data/blob/movement_direction_data_recovery_tools/atd-toolbox/direction_movement/recovery_environment/Dockerfile#L6).

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

**Comment by mateoclarke** [comment]
@frankhereford, I'm going to hold off on reviewing until you complete the steps you laid out in your comment above. I think you could exclude the last item about CRIS changes to `movement_id` and `travel_direction`. To me, that question falls under the scope of https://github.com/cityofaustin/atd-data-tech/issues/8746

**Comment by frankhereford** [comment]
This work has gotten to the point where it's produced a set of crashes for review by Xavier. I'm hopeful this is the final form, and if you all want to review it, I think this is the right time to do so.

Thanks!

**Review by chiaberry — APPROVED** [review]

**Review by patrickm02L — APPROVED** [review]
Rock and Roll!

**Comment by frankhereford** [comment]
Thank you for the reviews. It occured to me that prior to merging, I owe the `README.md` some documentation to explain the intent of this new toolbox entry. I will push that up as soon as I'm able and proceed to merging unless other feedback arises that I need to address. Again, thanks all!

File: atd-toolbox/direction_movement/docker-compose.yml
Comment (frankhereford) [inline atd-toolbox/direction_movement/docker-compose.yml:13]: This works fine, but it's obviously drawn from moped. I'll drop some more VZ oriented credentials in here. However, it's worth noting that anything works here, it's just confusing to have another, unrelated project's name used.

File: atd-toolbox/README.md
Comment (frankhereford) [inline atd-toolbox/README.md:50]: This is the actual new documentation related to the changes in this PR.

File: atd-toolbox/direction_movement/app/view_with_stakeholder_feedback_criteria.sql
Comment (frankhereford) [inline atd-toolbox/direction_movement/app/view_with_stakeholder_feedback_criteria.sql:15]: The values out of this column are copied into a scratch file and then piped back into the database to execute the restoration.

**Review by mateoclarke — COMMENTED** [review]
🙏  Thanks for the documentation in addition to the pretty easy to follow code. Hopefully the docker environment in the toolbox will be useful in the future too!

**Pull request merged**
